alpine-nix-rails-nginx/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# dkubb/alpine-nix-rails-nginx
+
+FROM dkubb/alpine-nix:latest
+MAINTAINER Dan Kubb <dkubb@fastmail.com>
+
+EXPOSE 80
+
+# Configure nix
+COPY config.nix /root/.nixpkgs/config.nix
+
+# Install system dependencies
+RUN nix-env --install --fallback \
+    acl-2.2.52 \
```

Comment:
What is the best way to check these "C implemnented dependencies" are upto date? I assume that nix imposes some restrictions in the versions we can use. Like packaging maintainers need to release updates etc. I'm not used to nix yet so I've to ask that question.